Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+VERSION ?= 0.0.2
```

Comment:
We should also add `  replaces: codeflare-operator.v0.0.1` the spec in the [csv template](https://github.com/project-codeflare/codeflare-operator/blob/main/config/manifests/bases/codeflare-operator.clusterserviceversion.yaml#L66).

Reply:
Maybe need something like:
```
# best if we can automatically detect the previous version
PREVIOUS_VERSION = 0.0.1
# then we can add a patch in the `bundle` target
cd config/manifests && $(KUSTOMIZE) edit add patch --patch '[{"op":"add", "path":"/spec/replaces", "value": "codeflare-operator.v$(PREVIOUS_VERSION)" }]' --kind ClusterServiceVersion
```

Reply:
Tracking this in a separate [issue](https://github.com/project-codeflare/codeflare-operator/issues/98)